data/json/mutations.json

```diff
@@ -3897,6 +3897,16 @@
         "prereqs" : ["ROT2"],
         "category": ["MUTCAT_ALPHA"],
         "healing_awake" : -0.0008
+    },{
+        "type" : "mutation",
+        "id" : "NO_HEALING_RESTING",
+        "name" : "Lost resting healing",
+        "points" : -2,
+        "starting_trait" : true,
+        "valid" : true,
+        "purifiable": true,
+        "description" : "Your default ability to regenerate during resting was lost.",
```

Comment:
Suggest `For whatever reason, you never seem to sleep deeply or effectively. Sleeping will not heal lost HP.`

Second sentence is uniform from slowhealer description to maintain conformity.